scripts/postbuild.ts

```diff
@@ -0,0 +1,26 @@
+import { readFileSync, readdirSync, renameSync, rmSync, writeFileSync } from 'node:fs'
+import { resolve } from 'node:path'
+
+function patchCjs(cjsModulePath: string, name: string) {
+  const cjsModule = readFileSync(cjsModulePath, 'utf-8')
+  writeFileSync(
+    cjsModulePath,
+    cjsModule.replace(`module.exports = ${name};`, `exports.default = ${name};`),
+    { encoding: 'utf-8' },
+  )
+}
+
+rmSync(resolve('./dist/index.d.mts'))
+rmSync(resolve('./dist/nuxt.d.mts'))
+
+readdirSync('./dist/shared')
+  .filter(file => file.endsWith('.d.mts'))
+  .forEach((file) => {
+    rmSync(resolve('./dist/shared', file))
+  })
+
+renameSync(resolve('./dist/index.mjs'), resolve('dist/index.js'))
+renameSync(resolve('./dist/nuxt.mjs'), resolve('dist/nuxt.js'))
```

Comment:
Why rename mjs to js? I'd actually prefer mjs to be explicit when possible

Reply:
We need to add the types (type is module):

![imagen](https://github.com/antfu/vite-plugin-inspect/assets/6311119/15e57ae9-7c0b-439d-9095-03e0e55a39cc)

Including types:

![imagen](https://github.com/antfu/vite-plugin-inspect/assets/6311119/d5ffe69c-86e5-4324-bcbb-204175ff67ce)